python/src/UniformOrderStatistics_doc.i.in

```diff
@@ -0,0 +1,50 @@
+%feature("docstring") OT::UniformOrderStatistics
+"UniformOrderStatistics distribution.
+
+.. warning::
+    This class is experimental and likely to be modified in future releases.
+    To use it, import the ``openturns.experimental`` submodule.
+
+Parameters
+----------
+dimension : int
+    The dimension of the distribution.
+
+See also
+--------
+openturns.MaximumEntropyOrderStatisticsDistribution
+
+Notes
+-----
+This is the joint distribution of the order statistics :math:`(X_{(1)},\dots,X_{(n)})` of a sample :math:`(X_1,\dots, X_n)` of :math:`n` independent random variables uniformly distributed over :math:`[0,1]`. Its probability density function is defined as:
```

Comment:
Splitting the text onto several lines will make the code easier to read.
Moreover, a small mathematical text will help guide users through the mathematical meaning of the class.
```suggestion
This is the joint distribution of the order statistics :math:`(X_{(1)},\dots,X_{(n)})`
of a sample :math:`(X_1,\dots, X_n)` of :math:`n` independent random variables
uniformly distributed over :math:`[0,1]`.
In other words:

- :math:`X_i \sim \mathcal{U}(0,1)` for :math:`i \in \set{1, ..., \inpuDim}` ;
- :math:`X_{(1)} \leq X_{(2)} \leq \ldots \leq X_{(\inputDim)}`.

Its probability density function is defined as:
```